src/markdown-pages/install-with-kurl/cis-compliance.md

```diff
@@ -0,0 +1,48 @@
+---
+path: "/docs/install-with-kurl/cis-compliance"
+date: "2021-12-14"
+weight: 24
+linktitle: "CIS Compliance"
+title: "CIS Compliance"
+isAlpha: false
+---
+The kURL installer can be configured to be Center for Internet Security (CIS) compliant. This is an opt-in feature that is configured in the kURL specification by setting the `kurl.cisCompliance` field to `true`.
+
+For information about known limitations, see [Known Limitations](#known-limitations).
+
+For more information about CIS security compliance for Kubernetes, see the [CIS benchmark information](https://www.cisecurity.org/benchmark/kubernetes).
+
+This YAML file example shows a valid specification for CIS compliance:
+
+```
+apiVersion: cluster.kurl.sh/v1beta1
+kind: Installer
+metadata:
+  name: cisCompliance
+spec:
+  kurl:
+    cisCompliance: true
+  kubernetes:
+    version: 1.19.15
+  containerd:
+    version: 1.4.6
+  antrea:
+    version: 1.4.0
+  rook:
+    version: 1.5.12
+  kotsadm:
+    version: 1.58.1
+  ekco:
+    version: 0.13.0
+  registry:
+    version: 2.7.1
+  velero:
+    version: 1.7.1
+```
+
+## Known Limitations
```

Comment:
@jala-dx We likely need something under known limitations about upgrading existing kURL to using this flag is unsupported. Below is first draft:

* Upgrading an existing kURL installation to including `kurl.cisCompliance` field to `true` is unsupported as the settings that are changed may introduce adverse effects.